README.md

```diff
@@ -1 +1,30 @@
-# MarkdownPreviewer
\ No newline at end of file
+# MarkdownPreviewer 🚀
+
+## Project Description 📝
+
+> This is the Markdown Previewer project created using React it's responsive and a freeCodeCamps frontend library project.
+
+- I learned how to create a react app.
```

Comment:
- Add what you learned specifically in React while working on this project for example:
  - I learned how to use and create React Components
  - I learned how to use React Hooks and especially `useState` to store the markdown previewer data on it
  - ...